### PR TITLE
generator: use <> instead of "" for Soletta headers inclusion

### PR DESCRIPTION
**Pull request by glima**
Signed-off-by: Gustavo Lima Chaves <gustavo.lima.chaves@intel.com>

### CODE REVIEW

File: src/bin/sol-fbp-generator/main.c [hunk 1]
```diff
@@ -1611,12 +1611,12 @@ generate(struct sol_vector *fbp_data_vector)
 
     /* Header name is currently inferred from the module name. */
     SOL_VECTOR_FOREACH_IDX (&ctx->headers, header, i) {
-        out("#include \"%.*s\"\n", SOL_STR_SLICE_PRINT(*header));
+        out("#include <%.*s>\n", SOL_STR_SLICE_PRINT(*header));
```
Comment (ibriano) [inline src/bin/sol-fbp-generator/main.c]: This one may be problematic, as some of the node types used could have been generated locally and not be installed.

**Comment by cabelitos** [comment]
Why are you changing that?

**Comment by glima** [comment]
Because the "" syntax tells the compiler to start searching for that at PWD, while the other tell its to start the search on system include dirs and -I entries, directly  (and that's the most common case when using the generator, anyway).

**Comment by glima** [comment]
https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html

**Comment by cabelitos** [comment]
@glima I know the difference between them. I just asked: "why the change that now?"

**Comment by glima** [comment]
Just being anal and maybe better at header lookups for the most common use case.

**Comment by cabelitos** [comment]
:new_moon_with_face:

**Comment by glima** [comment]
It seems I did not test it, meh. Dropping.

**Comment by bdilly** [comment]
It seems @ibriano is right about his assumptions.
So I suggest to drop this patch

**Pull request closed without merging**